solidity/contracts/test/keep/ECDSAKeepStub.sol

```diff
@@ -35,7 +36,7 @@ contract ECDSAKeepStub is IBondedECDSAKeep {
         // solium-disable-previous-line no-empty-blocks
     }
 
-    function distributeERC20Reward(address _asset, uint256 _value) external {
+    function distributeERC20Reward(IERC20 _asset, uint256 _value) external {
```

Comment:
This won't be enough, we have to play this out over both repos. Let's do that in a separate PR.

Reply:
> This won't be enough

We just need an interface change from `keep-ecdsa`.  This should be enough for TBTC

Reply:
> we have to play this out over both repos

Okay with moving to separate PR to unblock

Reply:
Well, the interface change will also require changing the implementors of the interface, etc. It's not going to be huge, just needs a little more finesse :)